From 0b1b0a4048b8e293b02fffe6414c0c9cb655529e Mon Sep 17 00:00:00 2001
From: "sukil.hong" <sukil.hong@lge.com>
Date: Fri, 21 Sep 2018 13:21:47 +0900
Subject: [PATCH] Enable ZRAM feature

:Release Notes:
Enable ZRAM feature

:Detailed Notes:
Change zram and zsmalloc configs

:Testing Performed:

:QA Notes:

:Issues Addressed:
[PLAT-64894] Find a way to apply swap feature in webOS ose
---
 arch/arm/configs/bcm2709_defconfig   | 4 ++--
 arch/arm64/configs/bcmrpi3_defconfig | 4 ++--
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/arch/arm/configs/bcm2709_defconfig b/arch/arm/configs/bcm2709_defconfig
index 63c8c10776b2..d78922fc4064 100644
--- a/arch/arm/configs/bcm2709_defconfig
+++ b/arch/arm/configs/bcm2709_defconfig
@@ -48,7 +48,7 @@ CONFIG_AEABI=y
 CONFIG_CLEANCACHE=y
 CONFIG_FRONTSWAP=y
 CONFIG_CMA=y
-CONFIG_ZSMALLOC=m
+CONFIG_ZSMALLOC=y
 CONFIG_PGTABLE_MAPPING=y
 CONFIG_UACCESS_WITH_MEMCPY=y
 CONFIG_SECCOMP=y
@@ -419,7 +419,7 @@ CONFIG_MTD_NAND=m
 CONFIG_MTD_SPI_NOR=m
 CONFIG_MTD_UBI=m
 CONFIG_OF_CONFIGFS=y
-CONFIG_ZRAM=m
+CONFIG_ZRAM=y
 CONFIG_BLK_DEV_LOOP=y
 CONFIG_BLK_DEV_CRYPTOLOOP=m
 CONFIG_BLK_DEV_DRBD=m
diff --git a/arch/arm64/configs/bcmrpi3_defconfig b/arch/arm64/configs/bcmrpi3_defconfig
index 772bedc916e3..4b2dd8061376 100644
--- a/arch/arm64/configs/bcmrpi3_defconfig
+++ b/arch/arm64/configs/bcmrpi3_defconfig
@@ -46,7 +46,7 @@ CONFIG_HZ_1000=y
 CONFIG_CLEANCACHE=y
 CONFIG_FRONTSWAP=y
 CONFIG_CMA=y
-CONFIG_ZSMALLOC=m
+CONFIG_ZSMALLOC=y
 CONFIG_PGTABLE_MAPPING=y
 CONFIG_SECCOMP=y
 CONFIG_ARMV8_DEPRECATED=y
@@ -406,7 +406,7 @@ CONFIG_MTD_BLOCK=m
 CONFIG_MTD_NAND=m
 CONFIG_MTD_UBI=m
 CONFIG_OF_CONFIGFS=y
-CONFIG_ZRAM=m
+CONFIG_ZRAM=y
 CONFIG_BLK_DEV_LOOP=y
 CONFIG_BLK_DEV_CRYPTOLOOP=m
 CONFIG_BLK_DEV_DRBD=m
-- 
2.17.1

